skopt/tests/test_space.py

```diff
@@ -160,6 +160,7 @@ def test_space_consistency():
     s3 = Space([Real(0, 1)])
     s4 = Space([(0.0, 1.0)])
     s5 = Space([(0.0, 1.0, "uniform")])
+    s6 = Space([(0, 1.0)])
```

Comment:
Well, according to the definition of ``Categorical``, those should be different because ``type(1) != type(1.0)``.